[2022-01-31 15:55:36,693] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-30T09:00:00+00:00 [queued]>
[2022-01-31 15:55:36,698] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-30T09:00:00+00:00 [queued]>
[2022-01-31 15:55:36,698] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 15:55:36,698] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2022-01-31 15:55:36,698] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 15:55:36,703] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2022-01-30T09:00:00+00:00
[2022-01-31 15:55:36,705] {standard_task_runner.py:54} INFO - Started process 18567 to run task
[2022-01-31 15:55:36,734] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2022-01-30T09:00:00+00:00', '--job_id', '21', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmpypq8rso9']
[2022-01-31 15:55:36,735] {standard_task_runner.py:78} INFO - Job 21: Subtask twitter_aluraonline
[2022-01-31 15:55:36,751] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-30T09:00:00+00:00 [running]> on host 1a222f561522
[2022-01-31 15:55:36,781] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2022-01-31 15:55:36,782] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-30T09:00:00.00Z&end_time=2022-01-31T09:00:00.00Z
[2022-01-31 15:55:36,801] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:55:37,118] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-30T09:00:00.00Z&end_time=2022-01-31T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe482idqxzo42rpid6savzg5dcq65
[2022-01-31 15:55:37,119] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:55:37,383] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20220130T090000, start_date=20220131T155536, end_date=20220131T155537
[2022-01-31 15:55:41,690] {local_task_job.py:102} INFO - Task exited with return code 0
